### PR TITLE
Group profile image upload

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

File: js/background.js [hunk 1]
```diff
@@ -750,7 +792,7 @@
       convo.updateGroup({
         groupId,
         groupName,
-        avatar,
+        nullAvatar,
```
Comment (Mikunj) [inline js/background.js]: Did you mean `avatar: nullAvatar` here?

Comment (neuroscr) [reply js/background.js]: yup

File: js/modules/loki_app_dot_net_api.js [hunk 2]
```diff
@@ -1256,6 +1289,15 @@ class LokiPublicChannelAPI {
     if (data.counts && Number.isInteger(data.counts.subscribers)) {
       this.conversation.setSubscriberCount(data.counts.subscribers);
     }
+
+    await window.Signal.Data.updateConversation(
```
Comment (neuroscr) [inline js/modules/loki_app_dot_net_api.js]: Does `this.conversation.set` not write to sqlite?

Comment (Mikunj) [reply js/modules/loki_app_dot_net_api.js]: No, i think it only writes to the backbone model.

**Pull request merged**